.github/workflows/prerequisite-test.yml

```diff
@@ -0,0 +1,14 @@
+name: Verify there is matching build of security plugin
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    name: Run prerequisite tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check the validation of OpenSearch Security Plugin
```

Comment:
The name of this should reflect what the check is performing. How does "Check for the existence of the OpenSearch Security Plugin artifact" sound?

Reply:
Sure, I have updated the commit and also changed the PR description with that. Thx for the review @cwperks!